.github/workflows/ci-onec-platform.yml

```
@@ -0,0 +1,76 @@
+      - uses: dorny/paths-filter@v3
+        id: changes
+        with:
+          filters: |
+            v8323: src/onec-platform/8.3.23.Dockerfile
+            v8324: src/onec-platform/8.3.24.Dockerfile
+            v8325: src/onec-platform/8.3.25.Dockerfile
+            v8326: src/onec-platform/8.3.26.Dockerfile
+            v8327: src/onec-platform/8.3.27.Dockerfile
+
+      - id: set-matrix
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:9:31: Double quote to prevent globbing and word splitting

(shellcheck)